src/controllers/technicalQualification.controller.js

```diff
@@ -2,10 +2,10 @@
 
 const Constants = require('../constants')
 const BaseController = require('./base.controller')
-const TechnicalQualification = require('../models/taskList/technicalQualification.model')
 const TechnicalQualificationValidator = require('../validators/technicalQualification.validator')
 const CookieService = require('../services/cookie.service')
 const Application = require('../models/application.model')
+const {WAMITAB_QUALIFICATION, REGISTERED_ON_A_COURSE, DEEMED_COMPETENCE, ESA_EU_SKILLS} = Constants.Dynamics.TechnicalQualification
```

Comment:
I think you need to remove the same definition of this from line 13.

Reply:
Done